doc/release/1.14.0-notes.rst

```diff
@@ -268,6 +282,10 @@ The new ``chebinterpolate`` function interpolates a given function at the
 Chebyshev points of the first kind. A new ``Chebyshev.interpolate`` class
 method adds support for interpolation over arbitrary intervals using the scaled
 and shifted Chebyshev points of the first kind.
+Support for reading lzma compressed text files in Python 3
```

Comment:
nit: line break here and below

Reply:
Yeah, I decided to leave off the rest of the release notes editing to just before the branch.

Reply:
I think all the formatting changes could do with collecting together coherently too.